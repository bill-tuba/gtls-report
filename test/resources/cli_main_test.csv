LN0,FN,a@b.c1,RED,1/1/1960
LNB,FN,a@b.c4,ORANGE,1/3/1990
LNB,FN,a@b.c3,YELLOW,1/1/1980
LN,FN,a@b.c4,GREEN,1/3/1990
LN,FN,a@b.c5,BLUE,1/3/1990
LNC,FN,a@b.c2,VIOLET,1/5/2000
LND,FN,a@b.c4,INDIGO,1/3/1990
FAIL,FN,a@b.c5,CRUD,1/3/1990
FAIL,FN,NOPE,BLUE,1/3/1990
FAIL,FN,a@b.c6,BLUE,xx
