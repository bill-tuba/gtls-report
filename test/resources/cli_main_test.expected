                                        

|	By email descending, last name ascending:
|
| LastName | FirstName |  Email | FavoriteColor | DateOfBirth |
|----------+-----------+--------+---------------+-------------|
|       LN |        FN | a@b.c5 |          BLUE |    1/3/1990 |
|       LN |        FN | a@b.c4 |         GREEN |    1/3/1990 |
|      LNB |        FN | a@b.c4 |        ORANGE |    1/3/1990 |
|      LND |        FN | a@b.c4 |        INDIGO |    1/3/1990 |
|      LNB |        FN | a@b.c3 |        YELLOW |    1/1/1980 |
|      LNC |        FN | a@b.c2 |        VIOLET |    1/5/2000 |
|      LN0 |        FN | a@b.c1 |           RED |    1/1/1960 |
                       

|	By birth date ascending:
|
| LastName | FirstName |  Email | FavoriteColor | DateOfBirth |
|----------+-----------+--------+---------------+-------------|
|      LN0 |        FN | a@b.c1 |           RED |    1/1/1960 |
|      LNB |        FN | a@b.c3 |        YELLOW |    1/1/1980 |
|      LNB |        FN | a@b.c4 |        ORANGE |    1/3/1990 |
|       LN |        FN | a@b.c4 |         GREEN |    1/3/1990 |
|       LN |        FN | a@b.c5 |          BLUE |    1/3/1990 |
|      LND |        FN | a@b.c4 |        INDIGO |    1/3/1990 |
|      LNC |        FN | a@b.c2 |        VIOLET |    1/5/2000 |
                       

|	By last name descending:
|
| LastName | FirstName |  Email | FavoriteColor | DateOfBirth |
|----------+-----------+--------+---------------+-------------|
|      LND |        FN | a@b.c4 |        INDIGO |    1/3/1990 |
|      LNC |        FN | a@b.c2 |        VIOLET |    1/5/2000 |
|      LNB |        FN | a@b.c4 |        ORANGE |    1/3/1990 |
|      LNB |        FN | a@b.c3 |        YELLOW |    1/1/1980 |
|      LN0 |        FN | a@b.c1 |           RED |    1/1/1960 |
|       LN |        FN | a@b.c4 |         GREEN |    1/3/1990 |
|       LN |        FN | a@b.c5 |          BLUE |    1/3/1990 |
